src/main/resources/templates/recipes/create.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body class="container">
+
+<header th:replace="fragments :: header"></header>
```

Comment:
This looks like it's based off of Katie's header for the ingredients page, so it's out of place on your page. Would recommend going into the fragments file and creating a new version for your recipes page.

Reply:
Pulling from fragments.html, Katie's looks like this:

`<nav th:fragment="header">`
`    <h1 th:text="${title}">Ingredients</h1>`
`    <ul class="nav">`
`        <li class="nav-item"><a class="nav-link" href="/ingredients">All Ingredients</a></li>`
`        <li class="nav-item"><a class="nav-link" href="/ingredients/add">Add Ingredient</a></li>`
`        <li class="nav-item"><a class="nav-link" href="/ingredients/delete">Delete Ingredients</a></li>`
`    </ul>`
`</nav>`

You can copy that if you like for your header -- it'll help maintain consistency.

Make sure when you copy it that you give it a proper name in the first line ("recipes-page-header" instead of just "header"), and change all the text and links so that they go to the proper parts of your recipes section.